demo/io.elementary.granite.demo.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id" : "io.elementary.granite.demo",
+    "runtime" : "io.elementary.Platform",
+    "runtime-version" : "daily",
+    "sdk" : "io.elementary.Sdk",
+    "command" : "granite-7-demo",
+    "tags" : [
+        "devel"
+    ],
+    "desktop-file-name-suffix" : " (Development)",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=network",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a",
+        "*.pyc",
+        "*.pyo"
+    ],
+    "modules" : [
+        {
+          "name" : "libsass",
```

Comment:
You use both 2 spaces and 4 spaces for indentation in this file. Please unify to 4 spaces.